HPXMLtoOpenStudio/resources/schedules.rb

```diff
@@ -1032,6 +1053,8 @@ def initialize(name, used_by_unavailable_periods, can_be_stochastic, type)
     Battery: Column.new('battery', false, false, :neg_one_to_one),
     BatteryCharging: Column.new('battery_charging', true, false, nil),
     BatteryDischarging: Column.new('battery_discharging', true, false, nil),
+    EVBatteryCharging: Column.new('ev_battery_charging', true, false, nil),
+    EVBatteryDischarging: Column.new('ev_battery_discharging', true, false, nil),
```

Comment:
Given where we ended up on these schedules, I think they should now be specified as fractions:
```suggestion
    EVBatteryCharging: Column.new('ev_battery_charging', true, false, :frac),
    EVBatteryDischarging: Column.new('ev_battery_discharging', true, false, :frac),
```